frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -121,7 +122,7 @@ public final class EnhancedGraphTable extends BaseEnhancedGraphTable
   private String mostRecentlyCheckedOutBranch = "?!@#$%^&";
 
   @UIEffect
-  private @Nullable Notification slideInNotification;
+  private @Nullable UnmanagedBranchNotification unmanagedBranchNotification;
```

Comment:
```suggestion
  private @MonotonicNonNull UnmanagedBranchNotification unmanagedBranchNotification;
```

Still compiles, but is more restrictive (only a non-null value can be assigned)